docs/Developers/Testing_coala_change_on_Bears.rst

```diff
@@ -0,0 +1,101 @@
+Testing changes to coala on coala-bears
+=======================================
+
+When large changes are made to the coala core, they should be tested
+on the coala-bears to confirm that they work as expected and don't
+break anything on the bears' side.
+
+You can do this by changing the coala dependency for the bears to
+be the one consisting of the changes. In order to do this, you need to create
+a new branch on coala-bears, update the coala requirement to point to your
```

Comment:
Changing `on coala-bears` to `on your coala-bears fork` will give contributor more information IMO